modules/nw-ovn-k-baseline-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,140 @@
+<4> Specify a name for the `ingress.name`
+<5> Specify the namespaces to select the pods from to apply the BANP resource.
+<6> Specify `podSelector.matchLabels` name of the pods to apply the BANP resource.
+====
+
+
+[discrete]
+[id="BaselineAdminNetworkPolicy-default-deny-example"_{context}]
+== BaselineAdminNetworkPolicy Deny example
+The following BANP singleton ensures that the administrator has set up a default deny policy for all ingress monitoring traffic coming into the tenants at `internal` security level. When combined with the "AdminNetworkPolicy Pass example", this deny policy acts as a guardrail policy for all ingress traffic that is passed by the ANP `pass-monitoring` policy.
+
+.Example YAML file for a guardrail `Deny` rule
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  name: default
+spec:
+  subject:
+    namespaces:
+      matchLabels:
+        security: internal
+  ingress:
+  - name: "deny-ingress-from-monitoring"
+    action: "Deny"
+    from:
+    - namespaces:
+        namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: monitoring
+# ...
+----
+====
+
+You can use an `AdminNetworkPolicy` resource with a `Pass` value for the `action` field in conjunction with the `BaselineAdminNetworkPolicy` resource to create a multi-tenant policy that allows one tenant to collect monitoring data on their application while simultaneously not collecting data from a second tenant.
+
+As an administrator if you apply both the "AdminNetworkPolicy `Pass` action example" and the "BaselineAdminNetwork Policy `Deny` example", tenants are then left with the ability to choose to create a `NetworkPolicy` resource that will be evaluated before the BANP.
+
+For example, Tenant 1 can set up the following `NetworkPolicy` resource to monitor ingress traffic:
+
+.Example `NetworkPolicy`
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-monitoring
+  namespace: tenant 1
+spec:
+  podSelector:
+  policyTypes:
+    - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: monitoring
+# ...
+----
+====
+
+In this scenario, tenant 1's policy would be evaluated after the "AdminNetworkPolicy `Pass` action example" and before the "BaselineAdminNetwork Policy `Deny` example", which denies all ingress monitoring traffic coming into tenants with `security` level `internal`. With Tenant 1's `NetworkPolicy` object in place, they will be able to collect data on their application while tenant 2 who does not have any `NetworkPolicy` objects in place will not be able to collect data, and as an administrator, you have not by default monitored internal tenants. Instead, you created a BANP that allows tenants to use `NetworkPolicy` objects to override the default behavior of your BANP.
```

Comment:
Tenant 1 is capitalized above but lower case here. 

tenant 2 is not capitalized. 

Does security need to be in backticks here? 

The second sentence probably needs broken up too. I'm having a problem rewriting the third part. so maybe consider: 

```suggestion
In this scenario, Tenant 1's policy would be evaluated after the "AdminNetworkPolicy `Pass` action example" and before the "BaselineAdminNetwork Policy `Deny` example", which denies all ingress monitoring traffic coming into tenants with `security` level `internal`. With Tenant 1's `NetworkPolicy` object in place, they will be able to collect data on their application. Meanwhile, Tenant 2, who does not have any `NetworkPolicy` objects in place, will not be able to collect data. As an administrator, you have not by default monitored internal tenants, but instead, you created a BANP that allows tenants to use `NetworkPolicy` objects to override the default behavior of your BANP.
```